folding-schemes/src/lib.rs

```diff
@@ -295,3 +307,28 @@ where
         proof: Self::Proof,
     ) -> Result<Vec<u8>, Error>;
 }
+
+pub trait SonobeField:
+    PrimeField<BasePrimeField = Self> + Absorb + AbsorbNonNative + Inputize<Self>
+{
+    type Var: FieldVar<Self, Self>;
+}
+
+impl<P: FpConfig<N>, const N: usize> SonobeField for Fp<P, N> {
+    type Var = FpVar<Self>;
+}
+
+pub trait SonobeCurve:
```

Comment:
How would you see naming them `Field` and `Curve` directly?
The reasoning I'm thinking is that it's shorter than `SonobeField` and `SonobeCurve`, which since it's a name that is used a lot across the code might be a bit tedious having always the prefix `Sonobe` there. And also that the 'Sonobe' prefix can be assumed since those traits are defined inside the Sonobe crate context.

Reply:
Makes sense! Will rename both.